app/views/home_page/index.html.erb

```diff
@@ -1,27 +1,29 @@
-<section class="banner-hero p-top-l p-bottom-l p-sides-m">
-  <div class="script-md banner-hero__txt"><%= I18n.t('root.hero') %></div>
-</section>
+<div class="content">
+  <section class="banner-hero p-top-l p-bottom-l p-sides-m">
+    <div class="script-md banner-hero__txt"><%= I18n.t('root.hero') %></div>
+  </section>
 
-<section class="banner-buttons p-top-m p-bottom-l p-sides-m">
-  <div class="p-bottom-m">
-    <h3 class="color-yellow"><%= I18n.t('action.search_by') %></h3>
-  </div>
-  <div class="button-holder">
-      <%= link_to I18n.t('action.month').capitalize, item_seasons_path, class: 'link_action button button--yellow script-lg rotate-minus-5' %>
-      <%= link_to I18n.t('action.produce').capitalize, produce_items_path, class: 'link_action button button--yellow script-lg rotate-zero' %>
-  </div>
-</section>
-
-<section class="banner-about p-top-m p-bottom-l">
-  <%= image_tag "https://dm2305files.storage.live.com/y4mYqMHpvb-2aKz6Q7P3jqr0EugNasCWn476P-zPwozPAPAxnKwadlz3S7DUa3FvDB2XLQdEI8rOvSkZTcITqKiIurRon4Fywldfm-p6DTsaNLzfXqXZUkFrkUVwxOswKTPKuMkElCHTjJZdNxSLwkr7bXSW3HyRvrI6J2MOl813zCFrm9YT9yP0pPecIaR9LZY?width=285&height=1217&cropmode=none", class: 'banner-about-img img-left' %>
-  <div class="banner-about__txt">
+  <section class="banner-buttons p-top-m p-bottom-l p-sides-m">
     <div class="p-bottom-m">
-        <h3 class="color-white"><%= I18n.t('root.about_title') %></h3>
+      <h3 class="color-yellow"><%= I18n.t('action.search_by') %></h3>
+    </div>
+    <div class="button-holder">
+        <%= link_to I18n.t('action.month').capitalize, item_seasons_path, class: 'link_action button button--yellow script-lg rotate-minus-5' %>
+        <%= link_to I18n.t('action.produce').capitalize, produce_items_path, class: 'link_action button button--yellow script-lg rotate-zero' %>
```

Comment:
**question:** why `link_to` over `normal `<a />`?

Reply:
what do you mean by `normal`?

Reply:
oh, sorry, I messed up the description. I meant a normal ` <a />`

Reply:
I was taught to use them as they were easier to maintain. Though that can be just a teachers opinion.

However, if I' m going to end using them in some more complicated instances, I find it easier to always use them. Instead of going between different ways to write the links.

Reply:
Sure, that's purely a matter of taste, I avoid any html helpers (and type the tags manually with whatever attributes would be rendered). Anyhow, please disregard my comment.